idd/Energy+.idd.in

```diff
@@ -33451,7 +33466,14 @@ ZoneHVAC:TerminalUnit:VariableRefrigerantFlow,
   A13,  \field Heating Coil Object Type
         \type choice
         \key Coil:Heating:DX:VariableRefrigerantFlow
+        \key Coil:Heating:DX:VariableRefrigerantFlow:FluidTemperatureControl
         \note Heating Coil Type must be Coil:Heating:DX:VariableRefrigerantFlow
+        \note if AirConditioner:VariableRefrigerantFlow is used 
+        \note to model VRF outdoor unit
+        \note Heating Coil Type must be 
+        \note Coil:Heating:DX:VariableRefrigerantFlow:FluidTemperatureControl
+        \note if AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl 
+        \note is used to model VRF outdoor unit
         \note This field may be left blank if cooling-only mode is used
   A14,  \field Heating Coil Object Name
         \type object-list
```

Comment:
\object-list HeatingCoilsDXVarRefrigFlowFluidTemperatureControl is added in IDD.